src/components/VideoHero/VideoHero.style.ts

```diff
@@ -1,26 +1,22 @@
 import { css } from '@emotion/react'
 import styled from '@emotion/styled'
-import { fluidRange } from 'polished'
 
 import { Button, IconButton, SkeletonLoader, Text } from '@/shared/components'
-import { breakpoints, colors, media, sizes } from '@/shared/theme'
+import { colors, media, sizes, typography } from '@/shared/theme'
 
 import { ChannelLink } from '../ChannelLink'
 
 const CONTENT_OVERLAP_MAP = {
-  SMALL: 25,
-  MEDIUM: 150,
-  LARGE: 200,
-  XLARGE: 400,
-  XXLARGE: 600,
+  SMALL: 24,
+  MEDIUM: 24,
+  LARGE: 24,
+  XLARGE: 24,
+  XXLARGE: 24,
 }
-const GRADIENT_OVERLAP = 50
-const GRADIENT_HEIGHT = 250
-const INFO_BOTTOM_MARGIN = 100
-const BUTTONS_HEIGHT_PX = '54px'
 
 export const Container = styled.section`
   position: relative;
+  max-height: 100vh;
 
   /* because of the fixed aspect ratio, as the viewport width grows, the media will occupy more height as well
```

Comment:
Why would we keep this section and all the styles below it? It's just the same style now repeated 5 times